### DISCUSSION TITLE
Auto-hide not working

### DISCUSSION THREAD

**Discussion by AllenW4L** [SnosMe/awakened-poe-trade#1179] [Q&A] [answered]
Since I update it I cant use the auto-hide feature when I move the mouse( Control + D), how to fix this ? my gpu is update, I reinstalled it and the game is windowed fullscreen. No vulkan. I deleted %appdata%\awakened-poe-trade. I still can search but I need to shift+space to close the search.

**Comment by SnosMe** [comment, marked as answer]
> Since I update it I cant use the auto-hide feature

There were 0 changes to that area of code for 6+ months, so search the issue in other place

**Reply by AllenW4L** [reply]
thanks... idk what to do
